### ISSUE TITLE
Missing accessibility

### ISSUE THREAD

**Issue by TTomczek** [TTomczek/studygroupmap#51]
Hello! I noticed that the study group map is missing some accessibility features. It would be great if we could add some more accessibility features to make the map more inclusive for everyone. Thank you!
